### PR TITLE
2018q4: Add news about :

### PR DESCRIPTION
**Pull request by evadot**
- DTS Update in head and 12-stable
- Marvell 8K support
- RockChip RK3399 support
- Pinebook image
- PWM subsystem

### CODE REVIEW

File: 2018q4/arm64-rockchip.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+## RockChip Support
+
+Contact: Emmanuel Vadot <manu@FreeBSD.org>
+
+Early support for the RockChip RK3399 have been commited.
```
Comment (debdrup) [inline 2018q4/arm64-rockchip.md]: s/have/has/

File: 2018q4/dts.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+## DTS Update
+
+Contact: Emmanuel Vadot <manu@FreeBSD.org>
+
+DTS files (Device Tree Sources) were updated to be in pair with Linux 4.20 for
+head and 4.19 for the 12-STABLE branch.
+
+The DTS are now compile for some arm64 boards as the one present in U-Boot are
```
Comment (debdrup) [inline 2018q4/dts.md]: Maybe I'm being too pedantic, but I think this should read "the DTS is now compiled for some arm64 boards, as..."

File: 2018q4/dts.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
+## DTS Update
+
+Contact: Emmanuel Vadot <manu@FreeBSD.org>
+
+DTS files (Device Tree Sources) were updated to be in pair with Linux 4.20 for
```
Comment (emaste) [inline 2018q4/dts.md]: on par with?

File: 2018q4/marvell.md [hunk 4]
```diff
@@ -0,0 +1,10 @@
+## Marvell 8K SoC support
+
+Contact: Emmanuel Vadot <manu@FreeBSD.org>
+Contact: Luis Octavio O Souza <loos@FreeBSD.org>
+
+Support for booting FreeBSD on Marvell 8K SoC (present on the MacchiatoBin for example)
+have been commited.
+As of today clocks, gpio, thermal, sdcard/eMMC drivers have been commited.
+SATA and USB were already working.
```
Comment (debdrup) [inline 2018q4/marvell.md]: I think there needs to be an extra newline here, to conform with the template.

**Comment by debdrup** [comment]
Merged, closing #44

**Pull request merged**